docs/integrations/sources/low-code.md

```diff
@@ -7,59 +7,59 @@ The changelog below is automatically updated by the `bump_version` command as pa
-| 0.88.1 | 2024-05-14 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.88.4 |
-| 0.88.0 | 2024-05-08 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.88.0 |
-| 0.87.0 | 2024-05-07 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.87.0 |
-| 0.86.2 | 2024-05-02 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.86.3 |
-| 0.86.1 | 2024-05-02 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.86.2 |
-| 0.86.0 | 2024-04-30 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.86.0 |
-| 0.85.0 | 2024-04-24 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.85.0 |
-| 0.84.0 | 2024-04-23 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.84.0 |
-| 0.83.1 | 2024-04-19 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.83.1 |
-| 0.83.0 | 2024-04-19 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.83.0 |
-| 0.82.0 | 2024-04-19 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.82.0 |
-| 0.81.8 | 2024-04-18 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.8 |
-| 0.81.7 | 2024-04-18 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.7 |
-| 0.81.3 | 2024-04-12 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.4 |
-| 0.81.2 | 2024-04-11 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.3 |
-| 0.81.1 | 2024-04-11 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.1 |
-| 0.81.0 | 2024-04-09 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.81.0 |
-| 0.80.0 | 2024-04-09 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.80.0 |
-| 0.79.2 | 2024-04-09 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.2 |
-| 0.79.1 | 2024-04-05 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.1 |
-| 0.79.0 | 2024-04-05 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.0 |
-| 0.78.9 | 2024-04-04 | [36834](https://github.com/airbytehq/airbyte/pull/36834) | Update CDK dependency to version 0.78.9 (before new publishing flow) |
+| Version | Date       | Pull Request                                                                                 | Subject                                                              |
+|:--------|:-----------|:---------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|
```

Comment:
@lazebnyi do you know why there's a version mismatch? isn't this file automatically updated?

Reply:
Note that there were a mismatch before it appeared in the changelogs as [this commit](https://github.com/airbytehq/airbyte/commit/86ddeb0a5dfd1c14581f1f0841f6cf42d8edcbec) shows the source-declarative-manifest version to be 2.1.0 while the CDK version was 2.3.0.

That being said, it seems to be [that change](https://github.com/airbytehq/airbyte/commit/2c2d841f5b93c8aaadf37163d88c6d2c2e858257#diff-4adf7de8a165844bb92806e52805e1a9f4f1755a32864c170d2ff89fd7f38c8cL20) that started the drift because the source-declarative-manifest was updated but not the CDK one. Maybe @bnchrch could chime in on that but I would assume it is a manual change that screwed the automatic updates

Reply:
The drift occured when `source-declarative-manifest` had code updates that were independant of the CDK

This was then compounded by the CDK updating its python version to one that was incompatible with SDM.

Leading to a number of SDM 3.x.x releases containing CDK4.x.x

This should be resolved now and were updating the tool so that a drift can no longer occur 
 https://github.com/airbytehq/airbyte/pull/42970